app/views/boards/show.haml

```diff
@@ -55,6 +55,11 @@
         %tr
           %td.continuity-spacer{colspan: 5}
       = render partial: 'posts/list_item', collection: @posts, as: :post, locals: { hide_continuity: true }
+
+    - if @board_sections.methods.include?(:total_pages) && @board_sections.total_pages > 1
```

Comment:
(I can't comment on the relevant line but) I think I remember there being a bug with board sections where if you stick `?page=2` in the URL then it'd show the second page or something weird like that, but I don't know why I would think that (we don't even paginate the posts inside sections right now, do we? it has `with_pagination: false`)

But just to be sure, when we use `?page=2` on a board with sections, it shows the second page of sections, but all the posts in those sections?

Reply:
I don't remember if I tested that particular case on the site! I'll look into that.

Reply:
Also it's been forever since I've last done Ruby but I think the normal way of asking this (the methods thing) is `.responds_to?(:total_pages)` - do we have this `.methods.include?` pattern somewhere else?

Reply:
(yes to the last question - https://discord.com/channels/521407923629457428/521411396613701662/1273042397470658746)